Makefile

```diff
@@ -134,7 +136,7 @@ gen:
 	./hack/update-docs.sh
 
 gen-docker:
-	$(CONTAINER_ENGINE) run --entrypoint make -it -v $(CURRENT_DIR):/go/src/sigs.k8s.io/descheduler -w /go/src/sigs.k8s.io/descheduler golang:1.22.3 gen
```

Comment:
What is `gen-docker` used for? Do we use it? Also, the golang image is missing a registry. Should `ocker.io/library/golang` be used as the default?

Reply:
some users reported having issues running gen, so this was a contribution to run it in docker.

it was added here https://github.com/kubernetes-sigs/descheduler/pull/1189/files

Reply:
I'm not sure if it is still used by anyone